frontend/src/pages/AlertDetails/hooks.tsx

```diff
@@ -277,6 +282,7 @@ export const useGetAlertRuleDetailsTimelineTable = (): GetAlertRuleDetailsTimeli
 			timelineFilter,
 			updatedOrder,
 			offset,
+			JSON.stringify(filters.items),
```

Comment:
When can this fail?

Reply:
@YounixM, IMO, it is not going to fail. if there are no filters, we're assigning `initialFilters` to filters.

https://github.com/SigNoz/signoz/blob/9e2386008ee52468c0ce0e49e10f3728bebdf503/frontend/src/constants/queryBuilder.ts#L159-L162

https://github.com/SigNoz/signoz/blob/9e2386008ee52468c0ce0e49e10f3728bebdf503/frontend/src/container/AlertHistory/Timeline/Table/Table.tsx#L16-L25

https://github.com/SigNoz/signoz/blob/9e2386008ee52468c0ce0e49e10f3728bebdf503/frontend/src/pages/AlertDetails/hooks.tsx#L257-L261